<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OCRWindow</class>
 <widget class="QMainWindow" name="OCRWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OCR</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>-10</y>
      <width>841</width>
      <height>91</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Menlo</family>
      <pointsize>40</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Optical Character Recognition (OCR)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="languageComboBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>420</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Menlo</family>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="inputPathButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Menlo</family>
     </font>
    </property>
    <property name="text">
     <string>Select Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="processButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>461</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Menlo</family>
     </font>
    </property>
    <property name="text">
     <string>Process Image</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="outputTextEdit">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>110</y>
      <width>441</width>
      <height>291</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Menlo</family>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="inputLabel">
    <property name="geometry">
     <rect>
      <x>47</x>
      <y>80</y>
      <width>461</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Menlo</family>
     </font>
    </property>
    <property name="text">
     <string>Input Preview</string>
    </property>
   </widget>
   <widget class="QLabel" name="outputLabel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>80</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Menlo</family>
     </font>
    </property>
    <property name="text">
     <string>Output Preview</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>410</y>
      <width>441</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Menlo</family>
     </font>
    </property>
    <property name="text">
     <string>Save Text</string>
    </property>
   </widget>
   <widget class="QPushButton" name="analyseButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>470</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Menlo</family>
     </font>
    </property>
    <property name="text">
     <string>Analyse Text</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ttsButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>470</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Menlo</family>
     </font>
    </property>
    <property name="text">
     <string>Speak out loud</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>461</width>
      <height>291</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Menlo</family>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="generateWordCloudButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>510</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Menlo</family>
     </font>
    </property>
    <property name="text">
     <string>Generate WordCloud</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
